spinn_front_end_common/interface/abstract_spinnaker_base.py

```diff
@@ -719,6 +714,12 @@ def run(self, run_time):
                 self._do_load()
                 loading_done = True
 
+        # Disable auto pause and resume if the binary can't do it
+        if (self._executable_start_type !=
+                ExecutableStartType.USES_SIMULATION_INTERFACE):
+            self._config.set("Buffers", "use_auto_pause_and_resume",
+                             "False")
```

Comment:
Not understanding why this has moved... surely this breaks things since the check for auto pause and resume has already been done...